pkgs/development/compilers/gcc/all.nix

```diff
@@ -34,9 +34,6 @@ let
               else            /* "4.8" */   isl_0_14;
       } // lib.optionalAttrs (majorMinorVersion == "4.8") {
         texinfo = texinfo5; # doesn't validate since 6.1 -> 6.3 bump
-      } // lib.optionalAttrs (majorMinorVersion == "4.9") {
-        # Build fails on Darwin with clang
-        stdenv = if stdenv.isDarwin then gccStdenv else stdenv;
```

Comment:
Please move this to another PR; it's an easy approval.  Also, that way it can go directly to `master`.

Reply:
Moved to https://github.com/NixOS/nixpkgs/pull/266194.